testing/openjfx11/APKBUILD

```diff
@@ -0,0 +1,270 @@
+# Contributor: Simon Frankenberger <simon-alpine@fraho.eu>
+# Maintainer: Simon Frankenberger <simon-alpine@fraho.eu>
+pkgname=openjfx11
+pkgver=11.0.3_p2
+_pkgver=${pkgver/_p/+}
+pkgrel=0
+pkgdesc="Open JavaFX 11"
+url="https://hg.openjdk.java.net/openjfx/11-dev/rt"
+arch="all !x86 !armhf !armv7" # openjdk11 is not yet available on 32 bit arches
+license="GPL-2.0 with Classpath"
+depends="$pkgname-gtk3" # for the virtual openjfx11 package
+makedepends="binutils-gold
+cmake
+ninja
+python2
+gperf
+openjdk11-jdk
+alsa-lib-dev
+ffmpeg-dev
+freetype-dev
+gtk+2.0-dev
+gtk+3.0-dev
+icu-dev
+ruby-dev
+sqlite-dev
+"
+subpackages="
+$pkgname-media:_media
+$pkgname-web:_web
+$pkgname-jmods:_jmods:noarch
+$pkgname-src:_src:noarch
+$pkgname-doc:_doc:noarch
+$pkgname-gtk2:_gtk2
+$pkgname-gtk3:_gtk3
+$pkgname-libs:_libs
+$pkgname-base:_base:noarch
+$pkgname-scenebuilder:_scenebuilder:noarch
+$pkgname-dev:_dev:noarch
+"
+source="openjfx-$_pkgver.tar.bz2::http://hg.openjdk.java.net/openjfx/11-dev/rt/archive/$_pkgver.tar.bz2
+https://archive.apache.org/dist/ant/binaries/apache-ant-1.8.2-bin.zip
+
+configure-ant-home.patch
+crosscompile.patch
+disable-architecture-verification.patch
+disable-assembler-on-unsupported-archs.patch
+disable-jit-for-non-x86.patch
+disable-sse2.patch
+disable-webkit-sampling-profiler.patch
+fix-arm32-build.patch
+fix-arm64-build.patch
+gcc-no-error.patch
+use-system-icu.patch
+webcore-linker-flags.patch
+webkit-debug-level.patch
+
+scenebuilder.bin
+"
+builddir="$srcdir/rt-$_pkgver"
+
+_ant_home="$srcdir/apache-ant-1.8.2/"
```

Comment:
Maybe we should make a `_ant_version` variable?

Reply:
I don't think it's needed here, the ant version is very unlikely to change (it has to be exactly this version and according to the history of this project it was defined in the beginning and hasn't changed since)

Reply:
Oh, alrighty then 👍